moped-api/claims.py

```diff
@@ -210,6 +216,8 @@ def put_claims(user_email: str, user_claims: dict, cognito_uuid: str = None):
             "user_id": {"S": user_email},
             "claims": {"S": encrypted_claims},
             "cognito_uuid": {"S": cognito_uuid},
+            "database_id": {"S": database_id},
+            "workgroup_id": {"S": workgroup_id}
```

Comment:
Thinking ahead, nobody asked me but I figured it could be useful if we need to make rules in Hasura about what groups can make certain changes

Reply:
Also glad that this came up because user workgroup has been floating around in my head since it came up a while back in a meeting. 

So, just so that I'm 100% sure, the database ID here is the `user_id` from the `moped_users` table, right?